pkgs/applications/virtualization/docker/default.nix

```diff
@@ -11,14 +11,14 @@ with lib;
 
 stdenv.mkDerivation rec {
   name = "docker-${version}";
-  version = "1.13.1";
-  rev = "092cba3"; # should match the version commit
+  version = "17.03.0-ce";
+  rev = "60ccb22"; # should match the version commit
```

Comment:
@copumpkin no problemo, it's a little hard to find and it's weird to have to do that to be honest 😅